botany-api/src/main/java/binnie/botany/api/IBotanyColored.java

```diff
@@ -0,0 +1,13 @@
+package binnie.botany.api;
+
+import net.minecraft.util.IStringSerializable;
+import net.minecraft.util.text.TextFormatting;
+
+import javax.annotation.Nullable;
+
+public interface IBotanyColored extends IStringSerializable {
```

Comment:
This is a useful interface, but it should not go in the `botany/api` directory unless it's needed by some other mod.
I think this can be kept internally in `botany` somewhere.

Reply:
Ok never mind, I see that it is needed by other classes in the API.
Ideally the logic for localization should be kept internal somehow, but it may be too difficult to untangle these classes right now.